src/searcher.rs

```diff
@@ -126,11 +142,13 @@ impl Searcher {
     ///         fields.
     ///     offset (Field, optional): The offset from which the results have
     ///         to be returned.
+    ///     order_by_order (Order, optional): The order in which the results
+    ///         should be sorted. If not specified, defaults to descending.
     ///
     /// Returns `SearchResult` object.
     ///
     /// Raises a ValueError if there was an error with the search.
-    #[pyo3(signature = (query, limit = 10, count = true, order_by_field = None, offset = 0))]
+    #[pyo3(signature = (query, limit = 10, count = true, order_by_field = None, offset = 0, order_by_order = Order::Desc))]
```

Comment:
Just checking my understanding: this must be `Desc` because the function it replaces, `order_by_u64_field` was also descending, according to these docs: https://docs.rs/tantivy/0.19.2/tantivy/collector/struct.TopDocs.html#method.order_by_u64_field

Is that correct?

Reply:
Yep, that's correct. It allows for code compatibility with the new version, which I defaulted to since there didn't seem to otherwise be a good reason to change the default.